user_guide_src/source/database/query_builder.rst

```diff
@@ -101,14 +101,14 @@ Example:
 
 .. literalinclude:: query_builder/005.php
 
-The first parameter enables you to set whether or not the query builder query
+The parameter in the first query (false) enables you to set whether or not the query builder
```

Comment:
I don't get what "the first query" is.

It means "the first parameter of the getCompiledSelect() method".
So "The first parameter" is correct and no problem.

Reply:
How about adding the default value for the parameter is `true`?